data/json/effects_on_condition/npc_eocs/hallucination_eocs.json

```diff
@@ -0,0 +1,195 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "hallucinate_effect",
+    "global": false,
+    "recurrence": [ "2 hours", "6 hours" ],
+    "condition": {
+      "and": [
+        { "not": { "u_has_effect": "sleep" } },
+        { "not": { "u_has_effect": "took_thorazine" } },
+        { "u_has_trait": "SCHIZOPHRENIC" }
+      ]
+    },
+    "effect": [
+      {
+        "weighted_list_eocs": [
+          [ "minor_hallucinations", { "const": 10 } ],
+          [ "intermediate_hallucinations", { "const": 6 } ],
+          [ "bad_hallucinations", { "const": 4 } ]
+        ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "minor_hallucinations",
+    "effect": [
+      {
+        "weighted_list_eocs": [
+          [ "paranoid", { "const": 75 } ],
+          [ "grandiose", { "const": 25 } ],
+          [ "self_talk", { "const": 14 } ],
+          [ "weapon_talk", { "const": 12 } ],
+          [ "weapon_talk_you_bleeding", { "const": 12 } ],
```

Comment:
Probably move this out of minor hallucinations into something that can happen whether you are having other experiences or not.  Otherwise the likelihood of it hitting at the same time you are bleeding is low.